lib/ansible/constants.py

```diff
@@ -178,6 +178,11 @@ def load_config_file():
                          'small', 'stegosaurus', 'stimpy', 'supermilker', 'three-eyes', 'turkey', 'turtle', 'tux', 'udder',
                          'vader-koala', 'vader', 'www',]
 
+PKG_MGR_MODULES = ['yum', 'dnf', 'apt', 'zipper', 'urpmi', 'pacman',
+                   'opkg', 'pkgin', 'macports', 'homebrew', 'apk',
+                   'pkgng', 'SD-UX', 'portage', 'svr4pkg', 'pkg5', 'xbps',
+                   'swupd', 'sorcery']
```

Comment:
For reasons unknown to me, openbsd_pkg is handle specifically in the pkg_mgr collection code and is left out of the list, so I left it that way. 

Jumping around git history a bit, and it looks like the special case for openbsd was introduced in 

```
commit f485eb28f53de8b8cda12ddc153cd5ef53f7c057
Author: Patrik Lundin <patrik.lundin.swe@gmail.com>
Date:   Sun Mar 3 00:02:50 2013 +0100

    Teach setup about OpenBSD package manager.
    
    * We cannot depend on the path because FreeBSD for example uses the same path.
```

The good thing is now that I extracted the 'list of pkg mgrs that have ansible modules and benefit from having the lock pre-created' from the 'list of pkg mgrs that guides the pkgmgr fact collection' I can add it to the former without breaking the latter._